tests/virtualization/vagrant/boxes/tumbleweed.pm

```diff
@@ -0,0 +1,144 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Test for vagrant and packaged addons
+# Maintainer: dancermak <dcermak@suse.com>
+
+use strict;
+use warnings;
+use base "basetest";
+use testapi;
+use utils;
+use File::Basename;
+
+#
+# Vagrantfile "template"
+#
+my $Vagrantfile_start = qq{Vagrant.configure("2") do |config|
+  config.vm.provider "virtualbox" do |virtbox|
+    virtbox.memory = 2048
+    virtbox.cpus = 4
+  end
+
+  config.vm.provider :libvirt do |libvirt|
+    libvirt.cpus = 4
+    libvirt.memory = 2048
+  end
+
+  config.vm.hostname = "vm-test"
+};
+
+my $Vagrantfile_end = qq {
+  config.vm.provision "shell", path: "provision.sh"
+
+  config.vm.provision :ansible do |ansible|
+    ansible.playbook = "ansible_playbook.yml"
+    ansible.become = true
+    ansible.become_user = "root"
+  end
+
+end
+};
+
+sub run() {
+    select_console('user-console');
+
+    # find the Vagrant boxes, they have the following filenames:
+    # openSUSE-Tumbleweed-Vagrant.x86_64-1.0-libvirt-Snapshot20190701.vagrant.libvirt.box
+    # openSUSE-Tumbleweed-Vagrant.x86_64-1.0-virtualbox-Snapshot20190701.vagrant.virtualbox.box
+
+    # FIXME: remove this
+    diag("autoinst_url" . autoinst_url);
+
+    foreach my $file (glob autoinst_url . "/*")                                    { diag("autoinst_url/*: " . $file); }
+    foreach my $file (glob autoinst_url . "/var/lib/openqa/factory/other/*") { diag(autoinst_url . "/var/lib/openqa/factory/other/*: " . $file); }
```

Comment:
globbing can't work because the access would happen over an HTTP URL. Did you try to `wget autoinst_url('assets/other/openSUSE-Tumbleweed-Vagrant.x86_64-1.0-libvirt-Snapshot20190703.vagrant.libvirt.box')` ?
And to not hard code you can expand variables. Take a look on the "Settings" tab of a test and you can see many variables available, e.g.:

```perl
my $version = get_required_var('VERSION');
my $arch = get_required_var('ARCH');
my build = get_required_var('BUILD');
autoinst_url("assets/other/openSUSE-$version-Vagrant.$arch-1.0-libvirt-Snapshot$build.vagrant.libvirt.box")` 
```

Reply:
Thanks for this suggestion, I've incorporated these changes.